<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="querryResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="findEmploye">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>650</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти сотрудника</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findPatient">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>650</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти Пациента</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="employeeInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="patientInput">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>650</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Фильтр поиска</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Фильтр поиска</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="employeeSurname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="employeeId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Id</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="employeeDepartment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Отделение</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="patientSurname">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>560</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="patientId">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>580</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Id</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="patientPolicy">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>600</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер полиса</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="employeePosition">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>560</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Должнось</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>20</width>
      <height>221</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>530</y>
      <width>20</width>
      <height>221</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1092</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
